cpp/models/abm/world.cpp

```diff
@@ -108,9 +92,32 @@ void World::migration(TimePoint t, TimeSpan dt)
                     if (wears_mask) {
                         person->migrate_to(target_location);
                     }
-                    break;
+                    return true;
                 }
             }
+            return false;
+        };
+
+        //run migration rules one after the other if the corresponding location type exists
+        //shortcutting of bool operators ensures the rules stop after the first rule is applied
+        if (m_use_migration_rules) {
+            (has_locations({LocationType::Cemetery}) && try_migration_rule(&get_buried)) ||
```

Comment:
Is there an advantage to use `has_locations` here instead of `has_location`?

Reply:
not really. consistency maybe. The compiler should (and does according to [this](https://godbolt.org/z/5WdMx8d7d) produce the same code since it's all transparent.